robot_calibration/include/robot_calibration/capture/depth_camera.h

```diff
@@ -37,21 +37,13 @@ class DepthCameraInfoManager
 
   bool init(ros::NodeHandle& n)
   {
-    camera_info_subscriber_ = n.subscribe("/head_camera/depth/camera_info",
+    std::string topic_name = n.getNamespace() +"/camera_info";
```

Comment:
I don't think this is doing what you think it is doing -- take a look at https://github.com/mikeferguson/robot_calibration/blob/indigo-devel/robot_calibration/src/ground_plane_finder.cpp#L34 -- this is the sort of thing you'll want to do